inginious/frontend/pages/tasks.py

```diff
@@ -256,8 +255,12 @@ def POST(self, courseid, taskid, isLTI):
                     if default_submissionid is None:
                         # This should never happen, as user_manager.update_user_stats is called whenever a submission is done.
                         return json.dumps({'status': "error", "text": _("Internal error")})
-
+                    
+                    # Adaptive hook call
+                    self.plugin_manager.call_hook("adaptive_post_hook", username=username, page=self, result=result)
```

Comment:
What's the purpose of the hook? My bet is that another name would be a little bit more self-descriptive. Remember that hooks must not be linked to a particular plugin, but to an INGInious feature to override.

Adding hooks specific to plugins is nothing more that coupling the code to the plugin.